src/titiler/xarray/tests/test_factory.py

```diff
@@ -135,22 +135,39 @@ def test_info_da_options(app):
     """Test /info endpoints with Dataarray options."""
     resp = app.get(
         "/md/info",
-        params={"url": dataset_4d_nc, "variable": "dataset", "drop_dims": "z=0"},
+        params={"url": dataset_4d_nc, "variable": "dataset", "sel": "time=2023-01-01"},
     )
     assert resp.status_code == 200
     assert resp.headers["content-type"] == "application/json"
+    body = resp.json()
+    print(body)
```

Comment:
nit: probably don't intend to keep the print statement here?

Reply:
😓 🙏